pkg/build/gobuild.go

```diff
@@ -639,14 +662,14 @@ func updatePath(cf *v1.ConfigFile) {
 		}
 		key, value := parts[0], parts[1]
 		if key == "PATH" {
-			value = fmt.Sprintf("%s:%s", value, appDir)
+			value = fmt.Sprintf("%s:%s", value, appPath)
```

Comment:
This would need a Windows-specific implementation to convert `appPath` to be suitable for `PATH`. In a quick test, generating a Windows image as described in the new README changes, I saw the generated `PATH` as this:

```bash
crane config <image> | jq .config.Env
## output
[
  "PATH=Files/ko-app/ko.exe",
  "KO_DATA_PATH=/var/run/ko"
]
```
Instead it would need to be something like:
```
[
  "PATH=c:\\ko-app\\ko.exe",
  "KO_DATA_PATH=/var/run/ko"
]
```
Note: While `PATH` would have to be Windows syntax, `KO_DATA_PATH` may not *need* to change since Go-generated Windows binaries seem to accept most POSIX-style paths, (though it would probably be more valid to do so).